inc/saiswitch.h

```diff
@@ -971,29 +971,32 @@ typedef enum _sai_switch_attr_t
      * @brief The hash object for IPv4 packets going through ECMP
      *
      * @type sai_object_id_t
-     * @flags READ_ONLY
+     * @flags CREATE_AND_SET
      * @objects SAI_OBJECT_TYPE_HASH
-     * @default internal
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
```

Comment:
not sure if there will be null instead of some hash set after switch create, maybe we could relax this add support "internal" as default value on non read_only object types